Makefile

```diff
@@ -29,8 +29,11 @@ SWAGGER_VERSION := v1.8.12
 STACKER := $(TOOLSDIR)/bin/stacker
 BATS := $(TOOLSDIR)/bin/bats
 TESTDATA := $(TOP_LEVEL)/test/data
-OS ?= linux
-ARCH ?= amd64
+#OS ?= $(shell case "$$(uname)" in (Darwin) echo "darwin";; (FreeBSD) echo "freebsd";; (*) echo "linux";; esac;)
```

Comment:
@rchincha , I've left two ways of detecting OS & ARCH (one is commented).
We should discuss which one is preferred.

Reply:
For the record, my choice would be to use `go env`

Reply:
remove the commented autodetect code and push.